readthedocs/settings/dev.py

```diff
@@ -48,6 +48,8 @@ def DATABASES(self):  # noqa
         'test:8000',
     )
 
+    SILENCED_SYSTEM_CHECKS = ['fields.W342', 'guardian.W001']
```

Comment:
Not sure that we need to duplicate this here, I believe the configs got merged with the ones from base

Reply:
When i tested this yesterday, it was not working. But somehow it is working fine now.
So, i have removed it from here.
Thanks for this.